test/dynamics/pulse/test_pulse_to_signals.py

```diff
@@ -122,6 +123,79 @@ def test_set_frequency(self):
         for idx in range(10):
             self.assertEqual(signals[0].samples[idx], np.exp(2.0j * idx * np.pi * -1.0 * 0.222))
 
+    def test_set_and_shift_frequency(self):
+        """Test that ShiftFrequency after SetFrequency is properly converted. It confirms implementation of phase accumulation is correct."""
+
+        duration = 20
+        dt = 0.222
```

Comment:
I defined variables often used here. We can find other frequently used number in other tests. So We can define these variables often used at setUp method.  I may refactor after merging this PR.